server/src/main/java/org/opensearch/action/admin/indices/tiering/TieringIndexRequest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.indices.tiering;
+
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.action.IndicesRequest;
+import org.opensearch.action.support.IndicesOptions;
+import org.opensearch.action.support.master.AcknowledgedRequest;
+import org.opensearch.common.annotation.ExperimentalApi;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Objects;
+
+import static org.opensearch.action.ValidateActions.addValidationError;
+
+/**
+ * Represents the tiering request for indices to move to a different tier
+ *
+ * @opensearch.experimental
+ */
+@ExperimentalApi
+public class TieringIndexRequest extends AcknowledgedRequest<TieringIndexRequest> implements IndicesRequest.Replaceable {
+
+    private String[] indices;
+    private Tier targetTier;
+    private IndicesOptions indicesOptions = IndicesOptions.fromOptions(false, false, true, false);
```

Comment:
Why does this need to be initialized here? 
nit: Also it would probably be more readable to have a default no-args constructor if it's really necessary.

Reply:
initialized it in a constructor rather